lib/terrafying/components/loadbalancer.rb

```diff
@@ -158,6 +157,10 @@ def attach(set)
         end
       end
 
+      def make_identifier(type, vpc_name, name)
+        "#{type[0..2]}-#{tf_safe(vpc_name)}-#{name}"[0..31]
```

Comment:
This will change the resource names for all LBs managed by this, which will cause them to all be replaced. 

Maybe just truncate the LB name field instead of also truncating the TF resource name?